Formula/s/swiftlint.rb

```diff
@@ -28,10 +28,16 @@ class Swiftlint < Formula
   uses_from_macos "libxml2"
 
   def install
-    args = if OS.mac?
-      ["--disable-sandbox"]
+    if OS.mac?
+      args = ["--disable-sandbox"]
     else
-      ["--static-swift-stdlib"]
+      libxml2_lib = Formula["libxml2"].opt_lib
+      args = [
+        "--static-swift-stdlib",
+        "-Xlinker", "-L#{Formula["curl"].opt_lib}",
+        "-Xlinker", "-L#{libxml2_lib}"
+      ]
+      ENV.prepend_path "LD_LIBRARY_PATH", libxml2_lib
```

Comment:
@carlocab I believe you mentioned this wasn't a good setting to use in brew. Could you have a look?